packaging/rpm/crc.spec

```diff
@@ -0,0 +1,226 @@
+# https://github.com/code-ready/admin-helper
+%global goipath         github.com/code-ready/crc
+Version:                1.22.0
```

Comment:
1.22.0 and 4.6.15 are hardcoded here. Should it be a template in the future?

Reply:
`Makefile` has `CRC_VERSION` and `BUNDLE_VERSION` so it's actually easy to avoid hardcoding them, I'll change it for `crc` (was harder in other modules). I'd prefer that we don't have `4.6.15` as part of the directory name/... though.

Reply:
They are no longer hardcoded in the latest version of the series.

Reply:
@cfergeau Have you already pushed that latest version of the series, I can still see the version info here.

Reply:
It's changed in a later commit `rpm: Don't hardcode version information in spec file`
https://github.com/code-ready/crc/pull/2014/commits/c2521e4df8d0d01c103e63c367f5bd88a884c2bf

Reply:
The `.spec` file has been renamed to `.spec.in`, the `.spec` file is autogenerated and contains a version.